src/app/server/CommissioningWindowManager.cpp

```diff
@@ -274,7 +277,7 @@ CHIP_ERROR CommissioningWindowManager::OpenBasicCommissioningWindow(Seconds16 co
     CHIP_ERROR err = OpenCommissioningWindow(commissioningTimeout);
     if (err != CHIP_NO_ERROR)
     {
-        Cleanup();
+        Cleanup(/* expireFailSafeTimer */ true);
```

Comment:
Again, I don't think that there's going to be a fail-safe armed at this point.  The OpenBasicCommissioningWindow command cannot be issued while a fail-safe is live, and opening a window at boot time or whatnot would not have a live fail-safe.